content/pages/volunteers.md

```diff
@@ -1,7 +1,7 @@
 Title: Volunteer
 
 **You can sign up for most volunteer positions by filling out [this
-form](http://goo.gl/forms/NsOXea6Yjf5t02Oj1).**
+form](https://lists.uwaterloo.ca/mailman/listinfo/wics-volunteers).**
```

Comment:
same issue with this old stuff (and you still have the file for the options panel in this diff too)